skyplane/__init__.py

```diff
@@ -18,6 +20,7 @@
 azure_config_path = config_root / "azure_config"
 azure_sku_path = config_root / "azure_sku_mapping"
 gcp_config_path = config_root / "gcp_config"
+clientid_config_path = config_root / "clientid_config"
```

Comment:
Can we just include this directly into SkyplaneConfig?

Reply:
I forgot to delete it. `clientid` is already included in `SkyplaneConfig` [here](https://github.com/skyplane-project/skyplane/blob/f474aaee0203d06497522ef73bd25c93f2512bf3/skyplane/config.py#L158-L160).